articles/tutorials/qft-tutorial.md

```diff
@@ -1,62 +1,77 @@
+* If you already have the QDK installed, make sure you have
+  [updated](xref:microsoft.quantum.update) to the latest version
 
 ## In this tutorial, you'll learn how to:
 
 > [!div class="checklist"]
+>
 > * Define quantum operations in Q#
-> * Set up the classical host program to call and simulate Q# operations
+>* Call and simulate Q# operations with the command line
+>  application for Q#.
 > * Simulate a quantum operation from qubit allocation to measurement output
-> * Observe how the quantum system's simulated wavefunction evolves throughout the operation
-
-Applications developed with Microsoft's Quantum Development Kit typically consists of two parts:
-1. A quantum program, implemented using the Q# quantum programming language, which is then invoked by the classical host program to run on a quantum computer or quantum simulator. These consist of 
-    - Q# operations: subroutines containing quantum operations, and 
-    - Q# functions: classical subroutines used within the quantum algorithm.
-2. A classical program, implemented in a classical programming language like Python or C#, that serves as the main entry point and will invoke Q# operations when it wants to execute a quantum algorithm.
+> * Observe how the quantum system's simulated wavefunction evolves throughout
+>   the operation
 
 ## Allocate qubits and define quantum operations
 
-The first part of this tutorial consists of defining the Q# operation `Perform3qubitQFT`, which performs the quantum Fourier transform on three qubits. 
+The first part of this tutorial consists of defining the Q# operation
+`Perform3QubitQFT`, which performs the quantum Fourier transform on three
+qubits.
 
-In addition, we will use the [`DumpMachine`](xref:microsoft.quantum.diagnostics.dumpmachine) function to observe how the simulated wavefunction of our three qubit system evolves across the operation.
+In addition, we will use the
+[`DumpMachine`](xref:microsoft.quantum.diagnostics.dumpmachine) function to
+observe how the simulated wavefunction of our three qubit system evolves across
+the operation.
 
-To create a your Q# project, follow these [instructions](xref:microsoft.quantum.howto.createproject), and create a Q# file entitled `Operations.qs`.
-We will walk you through the components of the file step-by-step, but the code is also available as a full block below.
+First you need to create a Q# project. If you don't know how to do it you can 
+take a look at the [installation guide](xref:microsoft.quantum.install.standalone).
 
 ### Namespaces to access other Q# operations
-Inside the file, we first define the namespace `Quantum.Operations` which will be accessed/imported by the classical host.
-For our operation to make use of existing Q# operations, we open the relevant `Microsoft.Quantum.<>` namespaces.
+
+Inside the file, we first define the namespace `Quantum.Operations` which will
+be accessed by the compiler. For our operation to make use of
```

Comment:
Definitely need to clarify that the namespace name is far less relevant (i.e. not at all relevant?) when using the command line application. 

I noticed that the project creation from templates (VS Code) automatically defines the namespace name to be whatever you named your project when you saved it. The point might be better mentioned in the install instructions, but I think we should also note here that the name isn't critical. The name we give it here, `Quantum.Operations` has such a jargon-y feel to it that I think newcomers might incorrectly assume it matters (worst case scenario: they've read some stuff about the difference between operations and functions, and now think that there might need to be a separate namespace, `Quantum.Functions`, if they were to include functions)